src/oracle/PCVOracle.sol

```diff
@@ -108,6 +135,34 @@ contract PCVOracle is IPCVOracle, CoreRefV2 {
         return (oracleValue * venueBalance) / Constants.ETH_GRANULARITY;
     }
 
+    /// @notice returns decimal normalized version of a given venues stale USD balance
+    /// does not account for unearned yield
+    function getVenueStaleBalance(address venue) public view returns (uint256) {
+        uint256 venueBalance = venueRecord[venue]
+            .lastRecordedBalance
+            .toUint256();
+
+        return venueBalance;
+    }
+
+    /// @notice returns decimal normalized version of a given venues USD pnl
+    function getVenueStaleProfit(
+        address venue
+    ) external view returns (uint256) {
```

Comment:
why is it needed in addition to `lastRecordedProfit(venue)` ? this seems incorrect because if a USDC venue has 1$ of profit, it has a `venueRecord[venue].lastRecordedProfit` of 1e18, and we return 1e18 * 1e30 / 1e18 = 1e30, which is $ 1,000,000,000,000 of profit